orion/tests/test_formatters.py

```
@@ -254,6 +257,264 @@ def test_format_average_produces_xml_element(self):
+
+    def test_comparison_table_with_none_pull(self):
+        periodic = _make_analysis_result()
+        pull_a = _make_pull_analysis([15.0])
+        pulls = [(1111, pull_a), (2222, None)]
+
+        table = _format_comparison_table(periodic, pulls)
+
+        assert "PR#1111" in table
+        assert "PR#2222" in table
+        lines = table.strip().split("\n")
+        data_line = [l for l in lines if "cpu" in l][0]
```

Ruff (0.15.14)

[warning] Prefer `next(l for l in lines if "cpu" in l)` over single element slice

Replace with `next(l for l in lines if "cpu" in l)`

(RUF015)